watertap/unit_models/crystallizer.py

```diff
@@ -558,6 +593,20 @@ def eq_crystallizer_height_constraint(b):
                 a + b + ((a - b) ** 2 + eps**2) ** 0.5
             )
 
+        # 13. Energy available from the vapor
+        @self.Constraint(doc="Thermal energy in the vapor")
+        def eq_vapor_energy_constraint(b):
```

Comment:
I'm not sure I understand this. It looks like the pure water and the vapor streams are exactly the same, or am I missing something? If they are not, how are they different?
- The flow rates are the same
- The pressures are the same